erpnext/education/doctype/quiz/quiz.py

```diff
@@ -59,3 +63,16 @@ def compare_list_elementwise(*args):
 	except TypeError:
 		frappe.throw(_("Compare List function takes on list arguments"))
 
+def evaluate_response(answers, response_dict):
```

Comment:
please drop docstrings and comments on what this function does, will make code review easier